File: stepik-union/src/main/java/org/stepik/plugin/refactoring/impl/idea/move/MoveHandlerDelegate.java
package org.stepik.plugin.refactoring.impl.idea.move;

import org.stepik.plugin.refactoring.impl.idea.AcceptableClasses;
import org.stepik.plugin.refactoring.move.AbstractMoveHandlerDelegate;

/**
 * @author meanmail
 */
public class MoveHandlerDelegate extends AbstractMoveHandlerDelegate {
    public MoveHandlerDelegate() {
        addAcceptableClasses(AcceptableClasses.get());
    }
}

File: stepik-union/src/main/java/org/stepik/plugin/refactoring/impl/idea/move/MoveHandlerDelegate.kt
package org.stepik.plugin.refactoring.impl.idea.move

import org.stepik.plugin.refactoring.impl.idea.AcceptableClasses
import org.stepik.plugin.refactoring.move.AbstractMoveHandlerDelegate


class MoveHandlerDelegate : AbstractMoveHandlerDelegate() {
    init {
        addAcceptableClasses(AcceptableClasses.get())
    }
}
